transport/TransportImpl.cpp

```diff
@@ -379,12 +379,22 @@ class UnregisteredJob : public jobmanager::CountedJob
 class RunTickJob : public jobmanager::CountedJob
 {
 public:
-    RunTickJob(TransportImpl& transport) : CountedJob(transport.getJobCounter()), _transport(transport) {}
+    RunTickJob(TransportImpl& transport, const uint64_t timestamp)
+        : CountedJob(transport.getJobCounter()),
+          _transport(transport),
+          _timestamp(timestamp)
+    {
+    }
 
-    void run() override { _transport.doRunTick(utils::Time::getAbsoluteTime()); }
+    void run() override
+    {
+        _transport._lastTickJobStartTimestamp = _timestamp;
```

Comment:
`_lastTickJobStartTimestamp` should be updated before post the new job on JobManager, otherwise it can be posted multiple times if smb starving on Worker threads

Reply:
removing _lastTickJobStartTimestamp and doing the check in EngineMixer resolves that and reduces iterating through all video transports.